bpf-filter/Makefile

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+
+# Frey: I decided to reuse the xdp-tutorial common framework. And here I
+#       assume that it is available in the same directory as this repository.
```

Comment:
Yeah, this assumption doesn't generally hold, obviously :)

We could re-create the whole libbpf subdir and build logic, but ultimately I think it may be easier to just move this whole thing to the bpf-examples repository :)

Reply:
Yes, exactly. At first, I thought of adding both libbpf and the commons directory from the xdp-tutorial to the repository. However, it just felt wrong. The makefiles in the xdp-tutorial are mainly tailored to that particular project and reference all eBPF code as XDP. Also, any beneficial changes to those files would then be out of sync with the xdp-tutorial files.

I will change these files and fix the scripts to cleanup when we run them the second time, like Jesper's scripts do. I have run my tests using virtme, which usually means that I rerun the VM instead of handling proper cleanup.

Reply:
Yeah, that's what the bpf-examples repo is for, basically. Started it because I got sick of copy-pasting the build environment all the time :D